hydrogym/firedrake/envs/cylinder/flow.py

```diff
@@ -200,7 +196,9 @@ def render(self, mode="human", clim=None, levels=None, cmap="RdBu", **kwargs):
 
 
 class RotaryCylinder(CylinderBase):
-  MAX_CONTROL = 0.5 * np.pi
+  MAX_CONTROL_LOW = -1.0
```

Comment:
Contradicts `MAX_CONTROL` recent update from https://github.com/dynamicslab/hydrogym/pull/195

Reply:
Yeah is this based on anything in particular?  If not, having it just match the pinball would probably make sense.